java/src/main/java/ai/rapids/cudf/ParquetColumnWriterOptions.java

```diff
@@ -220,22 +230,22 @@ public T withNullableTimestampColumn(String name, boolean isInt96) {
     public abstract V build();
   }
 
-  ParquetColumnWriterOptions(String columnName, boolean isTimestampTypeInt96,
+  public ParquetColumnWriterOptions(String columnName, boolean isTimestampTypeInt96,
                              int precision, boolean isNullable) {
```

Comment:
The motivation is that I don't think a builder is required for simple cases and forcing the user to use a builder is unnecessary IMHO e.g. a user just wants a String->String map. 

```
builder.withMapColumn(mapColumn("name", new ParquetColumnWriterOptions("key", false), new ParquetColumnWriterOptions("value", true)) 
```
instead of 
```
builder.withMapColumn(mapColumn("name", ParquetWriterOptions.builder().withColumns(false, "key").build(), ParquetWriterOptions.builder().withColumns(true, "value").build())) 
```

The user can still use the builder if they want

Reply:
I guess it's fine.  Please fix the indentation on the next line that wasn't updated when this line was.